src/command-processor.js

```diff
@@ -36,8 +36,18 @@ export async function processMessage(msg) {
                 return "I have recorded your progress friend.";
             case STATS_CMD:
                 const pastWeights = await weightStats(msg.author.username);
-                const average = await averageWeight(pastWeights);
-                return `Past 7 Weights: ${pastWeights}\n7 Day Average: ${average}`;
+                const reversedWeights = pastWeights.reverse()
```

Comment:
Do we need to reverse? The SQL query lists results in descending order by date so most recent should come first so you can slice the opposite way and calculate averages.

Reply:
I think it looks better reversed in discord but I will remove it for now.